js/modules/swiperCommentsFunctionality.js

```diff
@@ -1,74 +1,65 @@
-(function initializeClientsSpeakAboutUsSlider(){
-
-	const swiperComments = new Swiper(".swiper-container-2", {
-
-		navigation: {
-			nextEl: '.swiper_btn_next_2',
-			prevEl: '.swiper_btn_prev_2',
-		},
-		scrollbar: {
-			el: ".swiper_scrollbar_2",
-			draggable: true,
+import { setSlidesState } from "./swiperHelpers.js";
+import Swiper from "./swiper-bundle.8.4.5.esm.browser.min.js";
+
+const BTN_APPEARANCE_DELAY = 1000;
+const RESIZE_DELAY = 250;
+
+const swiperComments = new Swiper(".swiper-container-2", {
```

Comment:
swiper-container-2 is absolutely uninformative name for the class

Reply:
Done.